Rakefile

```diff
@@ -3,7 +3,11 @@ def type
 end
 
 def project_name
-  'Charts.xcodeproj'
+  'ChartsDemo/ChartsDemo.xcodeproj'
```

Comment:
I think we have merged master into 4.0 already? But these jumps out again

Reply:
Just haven't pulled from 4.0.0 in a while.